### ISSUE TITLE
syntax error in docstrings

### ISSUE THREAD

**Issue by bjornaa** [hetland/xroms#11]
Hi,

A trivial detail, but slightly annoying.

There are four syntax errors in xroms.py all related to the docstrings.
For roms_dataset, open_netcdf, open_zarr the docstring starts with ''' and end with """.
For the function hgrad is opposite, starts with """ and end with '''.

Automatic running of black before commit can be discussed,
but black (and possibly other tools) will prevent this kind of errors to get into the repo.

Bjørn Ådlandsvik

**Comment by kthyng** [comment]
Yes! Thank you. I am going to start improving the packaging. I haven't figured out how to run precommit checks and make improvements automatically, so if you know how to do that I would love the assistance.

**Comment by kthyng** [comment]
Hi again! I've recently put out a new version to PyPI (v0.2.4, still coming through on conda-forge). Please check to see if this issue is still present in the new version so that over time we can work to address these. Thanks!
